test/runtime/specs/juttle-spec/juttle-procs-join.spec.md

```diff
@@ -1079,3 +1079,65 @@ marked complete and participates in early joins. Later joins get the second tabl
     {"time":"1970-01-01T00:00:07.000Z","x":1,"z":8,"y":2}
     {"time":"1970-01-01T00:00:08.000Z","x":1,"z":9,"y":2}
     {"time":"1970-01-01T00:00:09.000Z","x":1,"z":10,"y":2}
+
+## batched join preserves mark calendarness of calendar intervals (join on shorter)
+
+### Juttle
+
+    (
+        emit -limit 2 -every :1M: -from :0: | put first=true, m=count() | batch :1M:;
+        emit -limit 4 -every :1M: -from :0: | put second=true, n=count() | batch :2M:
+    )
+    | join -outer 1
+    | view result -times true -marks true
+
+### Output
+
+    { "time": "1970-01-01T00:00:00.000Z", "interval": "1M", "mark": true }
```

Comment:
Ok, if we want to do this, we should be consistent across procs. I'll modify batch, reduce, fanin and join to emit the first mark without interval info. I'll put it into a separate commit, so it can be reverted later, if we decide against it.